File: src/main/java/org/arkadst/tempmotd/EventListener.java
package org.arkadst.tempmotd;

import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.server.ServerListPingEvent;

import java.util.List;

public class EventListener implements Listener {

    @EventHandler
    public void onPing(ServerListPingEvent event){
        List<String> motd = Main.config.getStringList("motd");
        event.motd(LegacyComponentSerializer.legacyAmpersand().deserialize(motd.get(0) + '\n' + motd.get(1)));
    }

}

File: src/main/kotlin/org/arkadst/tempmotd/EventListener.kt
package org.arkadst.tempmotd

import org.bukkit.event.server.ServerListPingEvent
import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener

class EventListener(private val main: Main) : Listener {

    @EventHandler
    fun onPing(event: ServerListPingEvent) {
        val motd = main.config.getStringList("motd")
        event.motd(LegacyComponentSerializer.legacyAmpersand().deserialize("${motd[0]}\n${motd[1]}"))
    }
}